node/chainSimulator/processorFactory.go

```diff
@@ -0,0 +1,23 @@
+package chainSimulator
+
+import (
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/process"
+)
+
+type processorFactory struct {
+}
+
+// NewProcessorFactory creates a new processor factory for normal chain simulator
+func NewProcessorFactory() ChainProcessorFactory {
```

Comment:
I don't think these name are right, since you create chain handlers, which return blocks creator

Reply:
Rename as they are used.

Reply:
renamed to `NewChainFactory`